packages/wallets/src/components/WalletText/WalletText.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import classNames from 'classnames';
+import styles from './WalletText.module.css';
+
+interface WalletTextProps {
+    align?: 'center' | 'left' | 'right';
+    children: React.ReactNode;
+    color?: 'black' | 'error' | 'general' | 'primary' | 'success' | 'warning' | 'white';
+    size?: '2xl' | '2xs' | '3xl' | '3xs' | '4xl' | '4xs' | 'l' | 'm' | 's' | 'xl' | 'xs';
```

Comment:
nit: can we sort according to ascending size?

Reply:
Tried 😄. But auto sort is making it this way.